pkg/parser/parser_test.go

```diff
@@ -3966,10 +4017,16 @@ func TestGetEndpoints(t *testing.T) {
 
 	for _, testCase := range tests {
 		t.Run(testCase.name, func(t *testing.T) {
-			result := getEndpoints(logrus.New(), testCase.svc, testCase.port, testCase.proto, testCase.fn)
+			result := getEndpoints(logrus.New(), testCase.svc, testCase.port, testCase.proto, testCase.fn, testCase.params)
 			if len(testCase.result) != len(result) {
 				t.Errorf("expected %v Endpoints but got %v", testCase.result, len(result))
 			}
+
+			for i := range result {
+				if result[i] != testCase.result[i] {
+					t.Errorf("expected %v to be equal to %v", result[i], testCase.result[i])
```

Comment:
```suggestion
					t.Errorf("expected %v but got %v", testCase.result[i], result[i])
```

make it clear which value was expected and which was received :)